### ISSUE TITLE
FFMPEG Issues

### ISSUE THREAD

**Issue by MikyMeen** [hjdhjd/homebridge-unifi-protect#1039]
### Homebridge UniFi Protect Version

v3.2.9

### Homebridge Platform and OS

macOS Catalina

### Homebridge Version

v1.7.0

### Node Version

v21.6.1

### UniFi OS Version

v3.2.9

### UniFi Protect Controller Version

General release

### Describe the problem

Since updating Unif OS and Protect to latest versions, homekit now needs FFMPEG. I have downloaded it as per the scripts in your support page but still no luck 

[1/25/2024, 9:51:48 AM] Homebridge v1.7.0 (HAP v0.11.1) (Homebridge 1158) is running on port 51546.
[1/25/2024, 9:51:51 AM] [UniFi Protect] Error running ffmpeg: Command failed: ffmpeg -hide_banner -version
dyld: Symbol not found: _VTRegisterSupplementalVideoDecoderIfAvailable
  Referenced from: /usr/local/bin/ffmpeg (which was built for Mac OS X 14.0)
  Expected in: /System/Library/Frameworks/VideoToolbox.framework/Versions/A/VideoToolbox

[1/25/2024, 9:51:51 AM] [UniFi Protect] Unable to probe the capabilities of your Homebridge host without access to 'ffmpeg'. Ensure that it is available in your path and correctly working.

### Homebridge HBUP JSON configuration

```json
{
    "controllers": [
        {
            "address": "192.168.0.1",
            "username": "MPTS HB",
            "password": "MPT3chS0luti0ns"
        }
    ],
    "name": "UniFi Protect",
    "platform": "UniFi Protect"
}
```

### Relevant log output

```shell
[1/25/2024, 9:51:48 AM] Homebridge v1.7.0 (HAP v0.11.1) (Homebridge 1158) is running on port 51546.
[1/25/2024, 9:51:51 AM] [UniFi Protect] Error running ffmpeg: Command failed: ffmpeg -hide_banner -version
dyld: Symbol not found: _VTRegisterSupplementalVideoDecoderIfAvailable
  Referenced from: /usr/local/bin/ffmpeg (which was built for Mac OS X 14.0)
  Expected in: /System/Library/Frameworks/VideoToolbox.framework/Versions/A/VideoToolbox


[1/25/2024, 9:51:51 AM] [UniFi Protect] Unable to probe the capabilities of your Homebridge host without access to 'ffmpeg'. Ensure that it is available in your path and correctly working.
```

### Acknowledgment that you are only running UniFi OS and UniFi Protect releases from the Ubiquiti Official release channel

- [X] I acknowledge that I am not running any early access / beta firmwares or operating systems from either Ubiquiti or Apple.

**Comment by hjdhjd** [comment]
Catalina is explicitly unsupported by ffmpeg-for-homebridge. Please see the [ffmpeg-for-homebridge](https://github.com/homebridge/ffmpeg-for-homebridge) for supported versions - currently Ventura or better. You can also roll your own on Catalina using Homebrew. [Instructions here](https://github.com/homebrew-ffmpeg/homebrew-ffmpeg/).

Quick view of instructions:
```
brew tap homebrew-ffmpeg/ffmpeg
brew install homebrew-ffmpeg/ffmpeg/ffmpeg --with-fdk-aac
```

TL;DR: you are running an unsupported macOS release. You will need to get your own working FFmpeg.
